sui/deploy-contract.js

```diff
@@ -269,16 +271,17 @@ async function upgrade(keypair, client, supportedPackage, policy, config, chain,
     const contractConfig = contractsConfig?.[packageName];
 
     validateParameters({ isNonEmptyString: { packageName } });
+    const moveDir = `${__dirname}/move`;
 
     if (packageDependencies) {
         for (const dependencies of packageDependencies) {
```

Comment:
need to split it by `,` or something to make it into an array?

Reply:
`packageDependencies` is  a [variadic options](https://github.com/tj/commander.js?tab=readme-ov-file#variadic-option), so we don't need to transform it manually.